src/core/packages/chrome/layout/core-chrome-layout-components/header/layout_header.styles.ts

```diff
@@ -12,7 +12,6 @@ import { EmotionFn } from '../types';
 
 const root: EmotionFn = ({ euiTheme }) => css`
   position: sticky;
-  overflow: hidden;
```

Comment:
I was checking the new layout in different browsers and noticed a bug in Safari. The push flyout from `EuiCollapsibleNavBeta` wasn't visible because of this. I'm removing it since it's not really needed, but taking it out will fix the bug in Safari.